procto.gemspec

```diff
@@ -17,5 +17,5 @@ Gem::Specification.new do |gem|
   gem.extra_rdoc_files = %w[LICENSE README.md CONTRIBUTING.md]
   gem.license          = 'MIT'
 
-  gem.add_development_dependency 'bundler', '~> 1.6.1'
+  gem.add_development_dependency 'bundler', '~> 1.11.2'
```

Comment:
So I am actually not clear on what the benefit of specifying the bundler version is in the first place. I'm happy to make the change but I'm curious if maybe the question is "do we even need this at all?"